datalad/core/local/tests/test_create.py

```diff
@@ -209,6 +209,15 @@ def test_create_sub(path):
     assert_in("third", ds.subdatasets(result_xfm='relpaths'))
 
 
+@with_tempfile
+def test_create_sub_gh3463(path):
+    ds = Dataset(path)
+    ds.create()
+    with chpwd(ds.path):
+        create("subds", dataset=ds.path)
```

Comment:
I know that this should be invocation identical to cmdline one... but I wonder if there could also be an explicit cmdline invocation e.g. via Runner just to make 100% sure?

Reply:
>  I wonder if there could also be an explicit cmdline invocation e.g. via Runner just to make 100% sure?

That's an overkill IMO, but I've update the test to do that.

Reply:
THANK YOU!  I will sleep better now ;)